README.md

```diff
@@ -347,7 +371,9 @@ _WARNING: This step requires the working OCI Registry, Cluster and Kyma CLI from
      target: control-plane
    ```
 
-2. Verify that the module creation succeeded and observe the `mod` folder. It will contain a `component-descriptor.yaml` with a definition of local layers.
+4. **Verification**
```

Comment:
This verification is not the fact anymore, CLI don't generate any `mod` folder, please change to the current way, maybe just mention the relevant part in the final `template.yaml`

Reply:
can you also verify `### Deploying a `ModuleTemplate` into the Control Plane`, is the description still up-to-date? Do we still need  `hack/local-template.sh`?

Reply:
The step `### Deploying Kyma infrastructure operators with `kyma alpha deploy`` is also outdated, it's deploy to `config/default` not to the control plane cluster.

Reply:
we still need the hack/local-template.sh file because k3d behaviour must still be circumvented. however, the script had to be updated.

Reply:
i am not sure if kyma alpha deploy is outdated. The config/default only lacks prometheus and grafana. Shoulf that be changed?

Reply:
ok, my bad, I understand this control plane cluster as using `control-plane` kustomize profile, should be fine with current discription